packages/mobile/src/onboarding/registration/PictureInput.tsx

```diff
@@ -0,0 +1,121 @@
+import colors from '@celo/react-components/styles/colors'
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Image, Platform, StyleSheet, Text, View } from 'react-native'
+import { TouchableOpacity } from 'react-native-gesture-handler'
+import ImagePicker from 'react-native-image-crop-picker'
+import OptionsChooser from 'src/components/OptionsChooser'
+import { Namespaces } from 'src/i18n'
+import Photo from 'src/icons/Photo'
+import Logger from 'src/utils/Logger'
+
+interface Props {
+  picture: string | null
+  onPhotoChosen: (photo: string) => void
+  backgroundColor: string
+}
+
+function PictureInput({ picture, onPhotoChosen, backgroundColor }: Props) {
+  const [showOptions, setShowOptions] = useState(false)
+  const updateShowOptions = (show: boolean) => () => setShowOptions(show)
+
+  const { t } = useTranslation(Namespaces.accountScreen10)
+
+  const pickPhoto = async (pickerFunction: typeof ImagePicker.openPicker) => {
+    try {
+      const image = await pickerFunction({
+        width: 200,
+        height: 200,
+        cropping: true,
+        includeBase64: true,
+        cropperCircleOverlay: true,
+        cropperChooseText: t('global:choose'),
+        cropperCancelText: t('global:cancel'),
+      })
+      // @ts-ignore
+      onPhotoChosen(`data:${image.mime};base64,${image.data}`)
```

Comment:
Does this mean we store the whole image in redux store?

Reply:
It does. From my tests images end up weighting around 27kb. The redux store was ~105kb big without the picture and jumped to ~132kb after saving it. I think it makes sense to leave it, but am open to other opinions on this.

Reply:
Can we store the photo using react-native-filesystem? The profile pictures of contacts will also need to be stored locally somewhere, and depending on how extensive their transaction history is, there might a lot of contacts.

Reply:
Saved it in the filesystem, just the path is now on the store :)